### DISCUSSION TITLE
Map doesn't update

### DISCUSSION THREAD

**Discussion by mebitek** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#399] [Q&A] [answered]
When I Start the robot, my map doesn't update. seems to work just for the first 5 seconds cause I see the robot icon moving and the first cleaning line. could please help me?

**Comment by PiotrMachowski** [comment, marked as answer]
https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor/issues/299
